functions-and-scope.js

```diff
@@ -53,12 +92,31 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+// ~*~ Waarom gaan jullie er steeds van uit dat ik geen herbruikbare functies schrijf? Dat is toch helemaal niet handig om te doen? Deze keer had ik toevallig al wel de goede naam bedacht.
 
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
+function averageGradeRounded(allGrades) {
+    let counter = 0;
+    let total = 0;
+
+    for (let i = 0; i <= (allGrades.length - 1); i++) {
+        counter = counter + 1;
+        total = total + allGrades[i];
+    }
+
+    const average = total/counter;
 
+    function roundToTwo(input) {
+        return +(Math.round(input + "e+2")  + "e-2");
```

Comment:
Dit kun je versimpelen:
```suggestion
        return input.toFixed(2);
```